src/main/java/io/jenkins/plugins/luxair/ImageTagParameterDefinition.java

```diff
@@ -94,6 +96,12 @@ public void setTagOrder(Ordering tagOrder) {
         this.tagOrder = tagOrder;
     }
 
+    public Boolean getVerifySsl() { return verifySsl; }
```

Comment:
```suggestion
    public Boolean getVerifySsl() { return verifySsl == null || verifySsl; }
```
Same as the earlier comment, if the job/pipeline configuration does not get re-saved then `verifySsl` is `null`. 
Finding this mad me realize that I messed up with my logic for the version ordering as well, but my fail did not throw a NPE due to how the value gets used in the `ImageTag` service, so I did not notice it till now.

Thanks to you I can submit a fix for my error and you can fix yours before this gets merged :smile: (no offense genuinely  happy I realized this now and hope u don't mid me noting this :wink: )